lua/neogit/popups/help/actions.lua

```diff
@@ -4,7 +4,7 @@ local util = require("neogit.lib.util")
 local NONE = function() end
 
 -- Using deep extend this way creates a copy of the mapping values
```

Comment:
I don't think this comment applies anymore

Reply:
Fair, will remove.